client/src/pages/Tree/Tree.js

```diff
@@ -56,62 +55,89 @@ export default function Tree({
 
   const handleClose = () => setCurrentTreeId(null);
   const imagePath = setFormatImagePath(scientific);
+  const [isMaintenanceAlertVisible, setMaintenanceAlertVisible] = useState(false)
+  const [maintenanceAlert, setMaintenanceAlert] = useState(null)
+
+  const handleMaintenanceAlert = (data) => {
+    setMaintenanceAlertVisible(true)
+    setMaintenanceAlert(data)
+
+    setTimeout(() => {
+      setMaintenanceAlert(null);
+      setMaintenanceAlertVisible(false);
+    }, 10000);
+  }
+
+  const closeMaintenanceAlert = () => {
+    setMaintenanceAlertVisible(false);
+    setMaintenanceAlert(null);
+  };
+
   return (
-    <TreeDetailsContainer
-      title="Tree Details"
-      width={drawerWidth}
-      open={!!currentTreeId}
-      onClose={handleClose}
-    >
-      {currentTreeData ? (
-        <>
-          <TreeHeader
-            currentTreeData={currentTreeData}
-            hasUnfitData={hasUnfitData}
-            isTreeQueryError={isTreeQueryError}
-          />
-
-          <ImageLoad src={imagePath} placeholder="placeholder.jpg" />
-
-          {!hasUnfitData && (
-            <TreeHealth
+    <div>
+      {isMaintenanceAlertVisible &&
+        maintenanceAlert
+      }
+      <TreeDetailsContainer
+        title="Tree Details"
+        width={drawerWidth}
+        open={!!currentTreeId}
```

Comment:
Hmm I did not add any changes to this section. This only appears because I added a div around the `TreeDetailsContainer` which indented this section. Maybe thats why the linter did not pick it up?